features/karaf/src/main/feature/feature.xml

```diff
@@ -244,6 +244,7 @@
   <feature name="esh-storage-mapdb" version="${project.version}">
     <feature>esh-base</feature>
     <bundle>mvn:org.eclipse.smarthome.storage/org.eclipse.smarthome.storage.mapdb/${project.version}</bundle>
+    <bundle dependency="true">mvn:org.mapdb/mapdb/1.0.9</bundle>
```

Comment:
FYI: I already used that for Guava as I stopped using the full-blown-mega swagger, jersey, ... bundles.
Xtext needs at least 10.0.1, swagger-jersey 18.0.

* https://github.com/eclipse/smarthome/blob/master/features/karaf-tp/src/main/feature/feature.xml#L173
* https://github.com/eclipse/smarthome/blob/master/features/karaf-tp/src/main/feature/feature.xml#L97

So, if you do not install the swagger-jersey stuff, you have installed 10.0.1 only, if you install it, you have only 18.0 (because Xtext can use both).